digiroad2-oracle/src/main/resources/db/migration/V1_40__fix_traffic_sign_enumerated_values.sql

```diff
@@ -0,0 +1,9 @@
+UPDATE enumerated_value
+SET name_fi = 'Väärä', modified_date = current_timestamp, modified_by = 'db_migration_v1_40'
+WHERE property_id = (SELECT id FROM property WHERE public_id = 'old_traffic_code')
+AND value = 0;
+
```

Comment:
Eikö noita nimiä voi korjata paremmiksi samalla vai heijastuuko se muuhun koodiin?

Reply:
Voisihan ne vaihtaa 'Tosi' ja 'Epätosi'. Nähdäkseni noita sanallisia selityksiä ei käytetä missään päin koodia.

Reply:
Tuo arvo voisi myös käsittääkseni olla mielummin tuolla single_choice_value taulussa, mutta en nyt kyllä viitsi tässä sitä refaktoroida